ui/imports/shared/controls/chat/MessageReactionsRow.qml

```diff
@@ -11,22 +11,25 @@ import SortFilterProxyModel
 Row {
     id: root
 
-    required property StatusEmojiModel emojiModel
-    required property var recentEmojis
-    required property string skinColor
+    required property SortFilterProxyModel emojiModel
```

Comment:
Hmm, this is very unusual to require specifically SortFilterProxyModel in the public API. Could it be used with just ListModel containing expected roles? The plain `var` is expected type for models.